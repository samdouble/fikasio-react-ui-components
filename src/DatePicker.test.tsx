import React from 'react';
import renderer from 'react-test-renderer';
import { DatePicker } from '.';

describe('DatePicker', () => {
  it('Renders correctly', () => {
    const componentJSON = renderer.create(<DatePicker />).toJSON();
    expect(componentJSON).toMatchSnapshot();
  });
});
