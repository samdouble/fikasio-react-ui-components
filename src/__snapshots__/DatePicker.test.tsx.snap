// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker Renders correctly 1`] = `
<div
  className="fikasio-datepicker fikasio-theme-dark"
  style={Object {}}
>
  <div>
    <div
      className="react-datepicker-wrapper"
    >
      <div
        className="react-datepicker__input-container"
      >
        <input
          className=""
          disabled={false}
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          readOnly={false}
          type="hidden"
          value=""
        />
      </div>
    </div>
  </div>
  <svg
    aria-hidden="true"
    className="svg-inline--fa fa-calendar-days fa-1x "
    data-icon="calendar-days"
    data-prefix="fas"
    focusable="false"
    onClick={[Function]}
    role="img"
    style={
      Object {
        "marginRight": 10,
      }
    }
    viewBox="0 0 448 512"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M128 0c17.7 0 32 14.3 32 32V64H288V32c0-17.7 14.3-32 32-32s32 14.3 32 32V64h48c26.5 0 48 21.5 48 48v48H0V112C0 85.5 21.5 64 48 64H96V32c0-17.7 14.3-32 32-32zM0 192H448V464c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V192zm64 80v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V272c0-8.8-7.2-16-16-16H80c-8.8 0-16 7.2-16 16zm128 0v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V272c0-8.8-7.2-16-16-16H208c-8.8 0-16 7.2-16 16zm144-16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V272c0-8.8-7.2-16-16-16H336zM64 400v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H80c-8.8 0-16 7.2-16 16zm144-16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H208zm112 16v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H336c-8.8 0-16 7.2-16 16z"
      fill="currentColor"
      style={Object {}}
    />
  </svg>
</div>
`;
